.gitignore

```diff
@@ -20,7 +20,10 @@ gcs_key_file.json
 # Folders for cmake/test output
 *_build/
 *_unity/
-*_unity_separate/
+*_unity_seperate/
```

Comment:
Thanks for sending this PR.  This is from some scripts, and also I believe it is spelt with an 'a'.  If you can undo this part though, should be good to add vscode

Reply:
@a-maurice seems as though the scripts are busted. Not really sure where that'd be at but I guess I can do a search/replace. The problem is that there's probably some folks that have already cloned the repo and are suddenly going to expect the output folder in one location and it might create a duplicate. This at least keeps everything tidy.